tests/test_getURL.py

```diff
@@ -0,0 +1,69 @@
+import unittest
+import ndio.remote.neurodata as neurodata
+import test_settings
+
+neurodata_token = "1524d74ec9a915536061af8fd5b51f2cde7de9a5"
+test_token = "b1876fab697e519e0cfe4acc9ca0496bf2889e17"
+Default_token = ""
+HOSTNAME = hostname=test_settings.HOSTNAME
+
+class TestgetURL(unittest.TestCase):
+
+    def setUp(self):
```

Comment:
Lets add the resource creation steps for all the different projects here. That way it runs reproducibly. Overall good though!